index.html

```diff
@@ -3,6 +3,9 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="./src/assets/logo.png" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;1,100&display=swap" rel="stylesheet">
```

Comment:
Poppins is already imported through the `base.scss` file. Please check on [here](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/blob/ft-login-page-185218916/src/styles/base.scss#L1)